search/routes/api/exceptions.py

```diff
@@ -6,14 +6,22 @@
 """
 
 from typing import Callable, List, Tuple
-
-from werkzeug.exceptions import NotFound, Forbidden, Unauthorized, \
-    MethodNotAllowed, RequestEntityTooLarge, BadRequest, InternalServerError, \
-    HTTPException
+from http import HTTPStatus
+
+from werkzeug.exceptions import (
+    NotFound,
+    Forbidden,
+    Unauthorized,
+    MethodNotAllowed,
+    RequestEntityTooLarge,
+    BadRequest,
+    InternalServerError,
+    HTTPException,
+)
 from flask import make_response, Response, jsonify
 
-from arxiv import status
```

Comment:
I agree with this removal; it's favorable to just use Werkzeug's HTTP exceptions instead.

Reply:
I saw the deprecation warning here: https://github.com/arXiv/arxiv-base/blob/master/arxiv/status/__init__.py#L13 , that's why I replaced the statuses.